src/components/App/App.component.jsx

```diff
@@ -1,57 +1,69 @@
-import React, { useLayoutEffect } from 'react';
-import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
+import React, { useState, useEffect } from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlay } from '@fortawesome/free-solid-svg-icons';
 import AuthProvider from '../../providers/Auth';
-import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
-import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
-import Layout from '../Layout';
-import { random } from '../../utils/fns';
+import VideoProvider from '../../VideoState/Provider';
+import ThemeContext from '../../providers/ThemeProvider/Provider';
+import AppTheme from '../AppTheme';
+
+const LoadingComponent = styled.div`
+  margin: auto;
+  text-align: center;
+  width: 50%;
+  padding: 10px;
+  margin-top: 20px;
+  font-size: 20px;
+  font-weight: bold;
+  span {
+    margin-left: 10px;
+  }
+`;
 
 function App() {
-  useLayoutEffect(() => {
-    const { body } = document;
+  const [gapiReady, setgapiReady] = useState(false);
 
-    function rotateBackground() {
-      const xPercent = random(100);
-      const yPercent = random(100);
-      body.style.setProperty('--bg-position', `${xPercent}% ${yPercent}%`);
-    }
+  useEffect(() => {
+    console.log('Component is mounted');
 
-    const intervalId = setInterval(rotateBackground, 3000);
-    body.addEventListener('click', rotateBackground);
+    if (!gapiReady) {
+      const script = document.createElement('script');
+      script.src = 'https://apis.google.com/js/client.js';
 
-    return () => {
-      clearInterval(intervalId);
-      body.removeEventListener('click', rotateBackground);
-    };
+      script.onload = () => {
+        window.gapi.load('client', () => {
+          window.gapi.client.setApiKey("AIzaSyAouhxcwwBMVFT6GoNF4ccP0_O7DnwmReY");
+          window.gapi.client.load('youtube', 'v3', () => {
+            setgapiReady(true);
+          });
+        });
+      };
+
+      document.body.appendChild(script);
+    }
   }, []);
 
+  if (gapiReady) {
+    console.log('ready!!');
+
+    return (
+      <BrowserRouter>
```

Comment:
You should replace for `HashRouter` because netlify server does not support different routes (doesn't send the same `index.html` for every route). If you go to any route and refresh the page you would end up on a 404.